### ISSUE TITLE
Padding error when decrypting video stream

### ISSUE THREAD

**Issue by imjuzcy** [DrmnSamoLiu/DrmnSamoLiu.github.io#2]
I'm also using a TP-Link IP camera (IPC42A-4 v5), and I stumbled upon your project while looking for a way to extract video from port 8800. My Wireshark capture looks almost identical to the one you shown on your website, but the continuation packet differs a little bit.

Mine:
```
----client-stream-boundary--
Content-Type: application/json
Content-Length: 100

{"type":"request","seq":0,"params":{"method":"get","preview":{"channels":[0],"resolutions":["HD"]}}}
```

When I ran your `video_stream.py` script, it ran past this stage, so I think the continuation packet difference doesn't matter, but when trying to decrypt the stream I got some error with the padding `[ERROR] Problem with padding` . 

According to your website it says 
```
Key : md5sum("admin":nonce)
IV : md5sum(nonce:passwd)
```

however, in the code, its the other way round. I have tried to change it, but still getting padding error. Can you confirm the script is still working?

**Comment by DrmnSamoLiu** [comment]
Hi, sorry for the late response.
Just to make sure first, are you using the password hash from config or your plaintext in the "passwd" field mentioned in the website?
Because in C200 the passwd needs to be hashed one, so I guess they do the same in their other products too.
I'm not quite sure right now but iirc the the passwd in config is hashed with md5sum too.

**Comment by imjuzcy** [comment]
Yes I'm using the hashed password and not the plain text password.
